modules/tinymce/src/themes/silver/main/ts/ui/general/OuterContainer.ts

```diff
@@ -196,16 +196,16 @@ const factory: UiSketcher.CompositeSketchFactory<OuterContainerSketchDetail, Out
       });
     },
     showMainView: (comp: AlloyComponent) => {
-      if (toolbarDrawerOpenState) {
-        apis.toggleToolbarDrawer(comp);
-      }
-
       Composite.parts.getPart(comp, detail, 'editorContainer').each((editorContainer) => {
         const element = editorContainer.element;
 
         Css.remove(element, 'display');
         Attribute.remove(element, 'aria-hidden');
       });
+
+      if (toolbarDrawerOpenState) {
```

Comment:
A some what hacky way to avoid the animation would be to first do toggle when it's hidden. Then do refresh when it visible that would re-measure things. The better way would be to add a option or separate function to toggleToolbarDrawer that immediately goes to the animation final state. Not sure what is animating that is it's sliding or transitioning but there should be a way to disable the animation for both.

Reply:
I did the hacky way to have something to merge in case we want this feature for the next release, but I'm digging to check how to get no animation

Reply:
I'll merge for now, maybe we can do a follow up to refactor it